src/commands/convert.ts

```diff
@@ -0,0 +1,63 @@
+import { promises as fPromises } from 'fs';
+import { flags } from '@oclif/command';
+import * as parser from '@asyncapi/parser';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { load } from '../models/SpecificationFile';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+import { convert } from '@asyncapi/converter';
+
+export default class Convert extends Command {
+  static description = 'convert asyncapi documents older to newer versions';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    file: flags.string({ char: 'f', description: 'path to the file to convert' }),
+    output: flags.string({ char: 'o', description: 'path to the file where the result is saved' }),
+    'target-version': flags.string({ char: 't', description: 'asyncapi version to convert to' }),
+    id: flags.string({ char: 'i', description: 'application id, if needed' })
+  }
+
+  static args = [
+    { name: 'spec-file', description: 'spec path, url, or context-name', required: false },
+  ]
+
+  async run() {
+    const { args } = this.parse(Convert);
```

Comment:
```suggestion
    const { args, flags } = this.parse(Convert);
```

Reply:
If you do this, you don't have to do `Convert.flags.` in the rest of the file. Also, not sure `Convert.flags.whatever` would give you the value of the flag. I think it would give you the definition of the flag above: https://github.com/asyncapi/cli/pull/188/files#diff-ab22990f7615d8bd1a90b1e9c264b48bd488123b5c36a2095c205e6b21086e72R13.